torch2jax/__init__.py

```diff
@@ -41,6 +41,9 @@ def RngPooperContext(value: RngPooper | None):
     _RNG_POOPER_STACK.pop()
 
 
+_GRAD_ENABLED = True
```

Comment:
let's add a comment here documenting why this exists and what parts of the api it interacts with